src/errors.js

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
 module.exports = {
```

Comment:
So, you can try to use const and arrow template 

const namePackage = 'QBMediaRecorder ';
'unsupport': `${namePackage} not supported this environment.`